app/helpers/applications_helper.rb

```diff
@@ -34,11 +34,20 @@ def formatted_application_data_value(key, value)
       project = Project.find_by_id value
       link_to_if project, project.try(:name), project
     else
-      auto_link simple_format(value)
+      auto_link(simple_format(value))
     end
     content_tag :p, formatted.html_safe
   end
 
+  def show_student_names(application)
+    output = ""
+    output << content_tag(:b, Application.data_label("student0_name"))
+    output << content_tag(:p, simple_format(application.team.students.first.name))
```

Comment:
I'm not working with Rails on a daily basis any more, but from what I remember, `simple_format` will already return a paragraph anyways. So you could reduce this to:
```ruby
# ...
output << content_tag(:b, Application.data_label("student0_name"))
output << simple_format(application.team.students.first.name)
# etc.
```